src/main/java/uk/gov/pay/adminusers/model/GetUserProfilesRequest.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.pay.adminusers.model;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+
+import java.util.List;
+
+import static com.google.common.collect.Lists.newArrayList;
+
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+public class GetUserProfilesRequest {
+
+    public static final String FIELD_EXTERNAL_IDS = "external_ids";
```

Comment:
Unless I’m missing something, this constant seems to be the only thing in this class that’s actually used anywhere. Do we actually need this class?